README.md

```
@@ -79,45 +83,55 @@ pnpm start:prod      # run production build
 pnpm test:cov        # test coverage
-pnpm lint            # eslint
+pnpm lint            # oxlint
+pnpm lint:fix        # oxlint --fix
+pnpm format          # oxfmt
+pnpm db:up           # start local PostgreSQL
+pnpm db:down         # stop local PostgreSQL
 ```
 
 ## Project structure
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)